src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -5059,34 +4988,19 @@ private bool ShouldModuleBeRemoved(PSModuleInfo module, string moduleNameInRemov
             return false;
         }
 
+        /// <summary>
+        /// Checks if an already loaded module meets the constraints passed to the module cmdlet by the user.
+        /// </summary>
+        /// <param name="alreadyLoadedModule">The already loaded module that matched the name of the module to load.</param>
+        /// <returns>True if the pre-loaded module matches all GUID and version constraints provided, false otherwise.</returns>
         internal bool IsModuleAlreadyLoaded(PSModuleInfo alreadyLoadedModule)
```

Comment:
This method has been in PowerShell with this name (and with this behaviour) since at least Win8. I would be happy to change it, so long as it's not going to cause any issues.